### PR TITLE
Feature/sass

### PR DESCRIPTION
**Pull request by domjngo**

### CODE REVIEW

File: css/sass/_header.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      position: absolute;
+      width: 1px;
+      height: 1px;
+      margin: -1px;
+      padding: 0;
+      overflow: hidden;
+      clip: rect(0,0,0,0);
+      border: 0;
+    }
+    input[type=submit] {
```
Comment (gtvj) [inline css/sass/_header.scss]: Just for consistency it's probably best to use either the quoted or non-quoted type attributes (both are currently used in the SASS)

File: Gruntfile.js [hunk 2]
```diff
@@ -2,10 +2,42 @@ module.exports = function (grunt) {
...
+                }
+            }
+        },
+        cssmin: {
+            options: {
+                sourceMap: true
+            },
+            target: {
+                files: {
+                    'css/blog-sass.css.min': ['css/blog-sass.css']
```
Comment (gtvj) [inline Gruntfile.js]: A minor point but I think the `-sass` suffix in the `.scss` files is a bit redundant. I'd change the file source file name to `blog.scss`, the compiled to `blog.css` and the minified to `blog.min.css`

File: inc/functions-blog.php [hunk 3]
```diff
@@ -16,9 +16,9 @@ function tna_child_styles()
 {
 	wp_register_style('tna-parent-styles', get_template_directory_uri() . '/css/base-sass.css.min', array(),
 		EDD_VERSION, 'all');
-	wp_register_style('tna-child-styles', get_stylesheet_directory_uri() . '/style.css', array(), '0.1', 'all');
+	wp_register_style('tna-child-sass', get_stylesheet_directory_uri() . '/css/blog-sass.css.min', array(), '0.1', 'all');
```
Comment (gtvj) [inline inc/functions-blog.php]: See comment above: `blog.css.min` would probably be better.

**Review by gtvj — CHANGES_REQUESTED** [review]
Hi @domingobishop - looks good. I've just suggested some minor changes for consistency in syntax and a file name change.

File: Gruntfile.js (hunk 2 above)
Comment (domjngo) [reply Gruntfile.js]: suffix removed

File: css/sass/_header.scss (hunk 1 above)
Comment (domjngo) [reply css/sass/_header.scss]: quotes added

File: inc/functions-blog.php (hunk 3 above)
Comment (domjngo) [reply inc/functions-blog.php]: suffix removed

**Review by gtvj — APPROVED** [review]

**Comment by gtvj** [comment]
Thanks @domingobishop - all merged now.

**Pull request merged**